x-pack/plugins/rule_registry/common/assets/field_maps/technical_rule_field_map.ts

```diff
@@ -14,14 +14,11 @@ export const technicalRuleFieldMap = {
     Fields.TIMESTAMP,
     Fields.EVENT_KIND,
     Fields.EVENT_ACTION,
-    Fields.RULE_UUID,
-    Fields.RULE_ID,
-    Fields.RULE_NAME,
-    Fields.RULE_CATEGORY,
     Fields.TAGS
   ),
-  [Fields.ALERT_OWNER]: { type: 'keyword', required: true },
-  [Fields.ALERT_PRODUCER]: { type: 'keyword' },
+  [Fields.ALERT_RULE_TYPE_ID]: { type: 'keyword', required: true },
+  [Fields.ALERT_RULE_CONSUMER]: { type: 'keyword', required: true },
+  [Fields.ALERT_RULE_PRODUCER]: { type: 'keyword' },
```

Comment:
Producer can be derived from the rule type id (`apm.error_count`) --> `apm` , so while convenient, we don't need to require it as long as we have the consumer and rule type id, we have enough for rbac.